cmd/iam.go

```diff
@@ -1096,11 +1096,6 @@ func (sys *IAMSys) IsAllowedServiceAccount(args iampolicy.Args, parentUser strin
 		return false
 	}
 
```

Comment:
You can't remove this @vadmeste because console adds empty JSON here. This is not correct..